tests/unit/test_telemetry_redaction.py

```
@@ -0,0 +1,16 @@
+from pydantic import SecretStr
+
+from flujo.utils.redact import summarize_and_redact_prompt
+from flujo.infra.settings import Settings
+
+
+def test_telemetry_pii_redaction(monkeypatch):
```

Ruff (0.14.7)

Unused function argument: `monkeypatch`

(ARG001)